### DISCUSSION TITLE
GET ops with token still hitting 10 request per minute ratelimit

### DISCUSSION THREAD

**Discussion by jodeepageAyaHC** [community/community#88586] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

From the documentation I read, when using a token to authenticate, the rate limit should be much higher. We have an internal website tool that queries from Github, LaunchDarkly and Jira to track our feature flags across time. It will pull flags for each team and then gather info from Jira and Github. The specific query to GH is a basic code search. When the page loads it will usually need to query anywhere from 5 to 50 flag strings. Authenticating with a token, but hitting rate limit after 10 calls. This is a very low rate limit and will prevent us from completing our tracking tool is a useful way. We could use the retry after new limit window is reset method, but that could take over 5 min to load one page if it has 50 flags to search....  Shouldn't the limit be higher when using a token??
![image](https://github.com/community/community/assets/147455414/157d93fc-c4aa-475c-8b93-11871e538329)

**Comment by ghostinhershell** [comment]
Hi @jodeepageAyaHC, 

Thanks so much for reaching out in the Community. We do have a post here in the Community, [Tips for avoiding the API rate limit](https://github.com/orgs/community/discussions/77255), that may help you out here. However, if you feel that you're incorrectly hitting the rate limit in your case, that would require further investigating. 

1. I recommend first [Checking the status of your rate limits](https://docs.github.com/en/rest/using-the-rest-api/rate-limits-for-the-rest-api?apiVersion=2022-11-28#checking-the-status-of-your-rate-limit) so that you have an idea of how many requests you're actually making within your rate limit window.
2. Is the token that you're using to authenticate being used to do any other API requests? This would also count against your rate limit.

If that still doesn't seem right, I recommend opening a ticket on our [Support page](https://support.github.com/contact) so that this can be investigated further.

**Comment by jodeepageAyaHC** [comment]
I have already reviewed those links. We are already at the least number of calls possible to accomplish the objective and I confirmed in the response headers that it is counting down from 10 max per minute despite using an access token. This is also a brad new tool with a newly created token so nothing else would be using it. I will go ahead and fill out a support ticket. Thank you!

**Comment by VevericAtomska** [comment]
❤️
